Makefile

```diff
@@ -1,2 +1,17 @@
+class:
+   maven compile -B -T 1C
```

Comment:
I believe Makefiles need to be indented with tabs

Reply:
Effectivly, atom just take the same indent as beffore